features/invoicing.feature

```diff
@@ -15,12 +15,11 @@ Feature: Invoicing
       """
       {
         "@context":"/api/contexts/Order",
-        "@id":"/api/orders",
+        "@id":"/api/invoice_line_items",
         "@type":"hydra:Collection",
         "hydra:member":[
           {
-            "@type":"Order",
-            "@id":@string@,
```

Comment:
@vladimir-8 is it a problem to not have the `@id` prop in response?

Reply:
It will break something in the UI unless we replace it with something else. Look for `@id` usages inside `js/app/admin/invoicing` directory

Reply:
Ok I will check how to have `@id` there